src/main/java/net/minestom/server/network/packet/server/common/TagsPacket.java

```diff
@@ -1,24 +1,36 @@
 package net.minestom.server.network.packet.server.common;
 
-import net.minestom.server.MinecraftServer;
-import net.minestom.server.gamedata.tags.Tag;
+import net.minestom.server.gametag.BlockTag;
+import net.minestom.server.gametag.EntityTag;
+import net.minestom.server.gametag.FluidTag;
+import net.minestom.server.gametag.GameEventTag;
+import net.minestom.server.gametag.ItemTag;
+import net.minestom.server.gametag.Tag;
 import net.minestom.server.network.NetworkBuffer;
-import net.minestom.server.network.packet.server.CachedPacket;
 import net.minestom.server.network.packet.server.ServerPacket;
 import net.minestom.server.network.packet.server.ServerPacketIdentifier;
-import org.jetbrains.annotations.ApiStatus;
+import net.minestom.server.utils.NamespaceID;
 import org.jetbrains.annotations.NotNull;
 
-import java.util.EnumMap;
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 import static net.minestom.server.network.NetworkBuffer.*;
 
 public record TagsPacket(
-        @NotNull Map<Tag.BasicType, List<Tag>> tagsMap) implements ServerPacket.Configuration, ServerPacket.Play {
-    @ApiStatus.Internal
-    public static final CachedPacket DEFAULT_TAGS = new CachedPacket(new TagsPacket(MinecraftServer.getTagManager().getTagMap()));
+        @NotNull Map<NamespaceID, List<Tag>> tagsMap) implements ServerPacket.Configuration, ServerPacket.Play {
+
+    public static @NotNull Map<NamespaceID, List<Tag>> createDefaultTags() {
+        Map<NamespaceID, List<Tag>> tagsMap = new HashMap<>();
+        tagsMap.put(NamespaceID.from(BlockTag.IDENTIFIER), new ArrayList<>(BlockTag.values()));
+        tagsMap.put(NamespaceID.from(ItemTag.IDENTIFIER), new ArrayList<>(ItemTag.values()));
+        tagsMap.put(NamespaceID.from(FluidTag.IDENTIFIER), new ArrayList<>(FluidTag.values()));
+        tagsMap.put(NamespaceID.from(EntityTag.IDENTIFIER), new ArrayList<>(EntityTag.values()));
+        // tagsMap.put(NamespaceID.from(GameEventTag.IDENTIFIER), new ArrayList<>(GameEventTag.values()));
```

Comment:
Why is this commented out?

Reply:
There is no registry and you get an exception